tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -1529,7 +1550,9 @@ async def complete(ls: ChapelLanguageServer, params: CompletionParams):
 
         items = []
         items.extend(
-            completion_item_for_decl(decl) for decl in fi.possibly_visible_decls
+            completion_item_for_decl(decl, override_name=name)
+            for (name, decl) in fi.visible_decls
+            if isinstance(decl, chapel.NamedDecl)
```

Comment:
I think this comprehension has reached a point of being more readable as an explicit loop